### ISSUE TITLE
grpcOptions not work in v1.0.2

### ISSUE THREAD

**Issue by zyf0330** [microsoft/etcd3#136]
I set grpcOptions with etcd3 v1.0.2, but it doesn't work. And I try 0.2.13, it works. I notice that v1.0.2 use @grpc/grpc-js, not sure if it is related to it.

**Comment by connor4312** [comment]
Please share your code and what you expect to happen / doesn't happen

**Comment by zyf0330** [comment]
I pass `grpcOptions` according to this doc https://microsoft.github.io/etcd3/interfaces/ioptions.html#grpcoptions.
This is my code.
```
new Etcd3({
    hosts:,
    auth: {
      username,
      password,
    },
    grpcOptions: {
      // reconnect
      'grpc.initial_reconnect_backoff_ms': 500,
      'grpc.min_reconnect_backoff_ms': 500,
      'grpc.max_reconnect_backoff_ms': 2000,
      // keepalive
      'grpc.http2.max_pings_without_data': 0,
      'grpc.http2.min_time_between_pings_ms': 10000, // default value of etcd server is 5s
      'grpc.keepalive_time_ms': 10000,
      'grpc.keepalive_timeout_ms': 5000,
      'grpc.keepalive_permit_without_calls': 1,
    },
  }
```
Keepalive options doesn't work in v1.0.2 but works in v0.2.13.

**Comment by connor4312** [comment]
These are the options grpc-js has, I think we need to update that doc to point here https://github.com/grpc/grpc-node/blob/21adc97c4ad458b9a09bd83c2cadc7fb2a345367/packages/grpc-js/src/channel-options.ts#L21-L39

**Comment by zyf0330** [comment]
I notice that some options doesn't appear in grpc-js options doc. Maybe it doesn't support them till now.
These options make etcd3 detect unreachable connection to etcd server fastly.

**Comment by connor4312** [comment]
If it's missing options, please open an issue on grpc-js; this library just wraps it, so any missing features will need to be implemented there.

**Comment by zyf0330** [comment]
I agree. But do you have any leisure to issue instead of me? I am afraid I cannot do this well.

**Comment by connor4312** [comment]
I would recommend for you to open the issue; you have the most context around what you need, and will also be able to verify that their additions work for your scenario once they add them.

**Comment by connor4312** [comment]
Closing this issue as it looks like it's upstream. Thanks for reporting it, though1

**Comment by zyf0330** [comment]
@connor4312 @grpc/grpc-js has released v1.2.0 which support more grpc options.

**Comment by zyf0330** [comment]
@connor4312 
I have tried these grpc options
```
    'grpc.http2.max_pings_without_data': 0,
    'grpc.http2.min_time_between_pings_ms': 10000, // default value of etcd server is 5s
    'grpc.keepalive_time_ms': 10000,
    'grpc.keepalive_timeout_ms': 5000,
    'grpc.keepalive_permit_without_calls': 1,
``` 
with both @grpc/grpc-js@v1.2.2 client and etcd3@1.1.0 which depends @grpc/grpc-js@v1.2.2.
I do tests by close network interface in ethernet.
The former connection becomes FIN-WAIT1 after timeout value 5 seconds after every 10 seconds keepalive probe, but the latter connection doesn't behavior same so doesn't detect network interruption.
